crates/fuel-gas-price-algorithm/gas-price-analysis/src/optimisation.rs

```diff
@@ -20,7 +22,8 @@ pub fn naive_optimisation(
         .map(|(p, d)| (run_simulation(*p, *d, avg_window), (*p, *d, avg_window)))
         .min_by_key(|(results, _)| {
             let SimulationResults { actual_profit, .. } = results;
-            actual_profit.iter().map(|p| p.abs()).sum::<i64>()
+            let err = actual_profit.iter().map(|p| p.abs()).sum::<i64>();
```

Comment:
I'm not following why we take the absolute value here? Are we interested in the total sum of the deviation between the reward and cost? Wouldn't we want an average here in that case?

Reply:
Nevermind, I misread this. We are never returning this value, as this is just used in `min_by_key` to find the parameters with the minimal profitability deviation.